docs/installation-windows.md

```diff
@@ -28,7 +28,7 @@ Alternatively, you can also download the individual scripts:
 
 - `scripts/windows/setup-wsl.ps1`
 - `scripts/windows/wsl-run.ps1`
-- `scripts/install-ubuntu1804.sh`
```

Comment:
I remember changes in concurrent https://github.com/aeternity/epoch/pull/1942 though shall not be major.

Reply:
it's ok after the rebase.